lua/wikis/commons/Widget/Match/HeaderBR.lua

```diff
@@ -0,0 +1,81 @@
+---
+-- @Liquipedia
+-- page=Module:Widget/Match/HeaderBR
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Lua = require('Module:Lua')
+
+local Array = Lua.import('Module:Array')
+local Class = Lua.import('Module:Class')
+local Operator = Lua.import('Module:Operator')
+
+local Widget = Lua.import('Module:Widget')
+local HtmlWidgets = Lua.import('Module:Widget/Html/All')
+local Trophy = Lua.import('Module:Widget/Match/Summary/Ffa/Trophy')
+local Div = HtmlWidgets.Div
+
+local OpponentDisplay = Lua.import('Module:OpponentDisplay/Custom')
+local Placement = Lua.import('Module:Placement')
+
+---@class MatchHeaderBRProps
+---@field match FFAMatchGroupUtilMatch
+---@field teamStyle? teamStyle
+
+---@class MatchHeaderBR: Widget
+---@operator call(MatchHeaderBRProps): MatchHeaderBR
+---@field props MatchHeaderBRProps
+local MatchHeaderBR = Class.new(Widget)
+MatchHeaderBR.defaultProps = {
+	teamStyle = 'short',
+}
+
+---@return Widget?
+function MatchHeaderBR:render()
+	local match = self.props.match
+	if not match then
+		return nil
+	end
+
+	if #match.opponents <= 2 or not match.finished then
+		return
+	end
+
+	local topThree = Array.filter(
+		Array.sortBy(match.opponents, Operator.property('placement')),
+		function(opponent) return opponent.placement <= 3 end
+	)
```

Comment:
This might still return more than three opponents

Reply:
You get that issue regardless of where you place the cutoff - there could the 4 opponents on tied first for example.
I think this is fine, as it won't break the display by showing more than 3 opponents, and doesn't display wrong placements by using the actual opponent placement.